module/VuFind/src/VuFind/Search/UrlQueryHelper.php

```diff
@@ -568,13 +568,29 @@ public function setLimit($l)
         );
     }
 
+    /**
+     * Return HTTP parameters to render the current page with a different jumpto
+     * parameter.
+     *
+     * @param bool $jumpto If results page is skipped when a search has only one hit
```

Comment:
Should this be `false|int`? While setting $jumpto to false (or 0) will disable it, setting it to true will potentially cause search results to jump to the first result regardless of the result set size. I think it might be misleading to treat this purely as a boolean setting.

You're also welcome to add types to the method definition itself if you wish; I realize the legacy code is all untyped, but that doesn't prevent us from doing better on new things we add. :-)